ledger-api/grpc-definitions/com/daml/ledger/api/v2/command_service.proto

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v2;
+
+import "com/daml/ledger/api/v2/commands.proto";
+import "com/daml/ledger/api/v2/transaction.proto";
+import "google/protobuf/empty.proto";
+
+option java_outer_classname = "CommandServiceOuterClass";
+option java_package = "com.daml.ledger.api.v2";
+option csharp_namespace = "Com.Daml.Ledger.Api.V2";
+
+// Command Service is able to correlate submitted commands with completion data, identify timeouts, and return contextual
+// information with each tracking result. This supports the implementation of stateless clients.
+//
+// Note that submitted commands generally produce completion events as well, even in case a command gets rejected.
+// For example, the participant MAY choose to produce a completion event for a rejection of a duplicate command.
+service CommandService {
+  // Submits a single composite command and waits for its result.
+  // Propagates the gRPC error of failed submissions including Daml interpretation errors.
+  rpc SubmitAndWait (SubmitAndWaitRequest) returns (google.protobuf.Empty);
+
+  // Submits a single composite command, waits for its result, and returns the transaction id.
+  // Propagates the gRPC error of failed submissions including Daml interpretation errors.
+  rpc SubmitAndWaitForTransactionId (SubmitAndWaitRequest) returns (SubmitAndWaitForUpdateIdResponse);
```

Comment:
There is Update in the response but elsewhere in this service we only speak about transactions. Is it a generic service (transaction+reassignments) or only transactions?

Reply:
this is meant to be kept as is: transactions only.
thanks for pointing out, fixed couple of missing renames here